test/ng/sceSpecs.js

```diff
@@ -437,7 +437,7 @@ describe('SCE', function() {
           runTest({whiteList: ['http://***']}, null)();
         }).toThrowMinErr('$injector', 'modulerr', new RegExp(
              /Failed to instantiate module function ?\(\$sceDelegateProvider\) due to:\n/.source +
-             /[^[]*\[\$sce:iwcard\] Illegal sequence \*\*\* in string matcher\.  String: http:\/\/\*\*\*/.source));
+             /[^[]*\[\$sce:iwcard\] Illegal sequence \*\*\* in string matcher\. {2}String: http:\/\/\*\*\*/.source));
```

Comment:
It didn't shout at me so I haven't.

On Mon, Aug 1, 2016 at 2:41 PM, Georgios Kalpakas <notifications@github.com>
wrote:

> In test/ng/sceSpecs.js
> <https://github.com/angular/angular.js/pull/14952#discussion_r72970773>:
>
> > @@ -437,7 +437,7 @@ describe('SCE', function() {
> >            runTest({whiteList: ['http://***']}, null)();
> >          }).toThrowMinErr('$injector', 'modulerr', new RegExp(
> >               /Failed to instantiate module function ?\(\$sceDelegateProvider\) due to:\n/.source +
> > -             /[^[]*\[\$sce:iwcard\] Illegal sequence \*\*\* in string matcher\.  String: http:\/\/\*\*\*/.source));
> > +             /[^[]*\[\$sce:iwcard\] Illegal sequence \*\*\* in string matcher\. {2}String: http:\/\/\*\*\*/.source));
>
> How come you didn't unescape \]? 😃
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.js/pull/14952/files/a35c7b1049edc8033c0e3a6eaeeda6cd34c99e73#r72970773>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABrUnkJmhhYGfHEGwaNC3ZMkUqCoKCnqks5qbemIgaJpZM4JU90D>
> .
>



-- 
Michał Gołębiowski